backend/src/main/resources/db/migration/V11__move_address_to_parcel.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE parcels
+    ADD COLUMN address_id UUID;
+
+ALTER TABLE parcels
+    ALTER COLUMN address_id SET NOT NULL;
```

Comment:
```suggestion
UPDATE parcels
  SET address_id = recipients.address_id
  FROM recipients
  WHERE parcels.recipient_id = recipients.id
  AND recipients.address_id IS NOT NULL;

ALTER TABLE parcels
    ALTER COLUMN address_id SET NOT NULL;
```

I have two suggestions here.

First, we need to add some data to the address columns of the parcels before we set them as `NOT NULL`. Otherwise, it might throw out errors.

Second, to avoid losing data from previous entries, we should copy the addresses already in the recipients table into the parcel table.